cmake/tpl/polytope.cmake

```diff
@@ -1 +1,4 @@
 set(${lib_name}_libs libpolytope.a)
+
+include(${SPHERAL_ROOT_DIR}/cmake/spheral/SpheralHandleExt.cmake)
```

Comment:
Includes for TPL functions can probably be moved to `cmake/InstallTPLs.cmake` so we don't need to keep redefining them.

Reply:
Ok, I've moved them to that file.